src/js/factual_background.js

```diff
@@ -234,7 +246,30 @@ class FactualBackground {
     }
 
     if (tabInfo.isContentLoaded()) {
+      // TODO to check: this might stop checking content on newly loaded Facebook posts
```

Comment:
Czy to `TODO` jest do powyższego `if`a? Jeśli tak, to on był potrzebny przy poprzednim tasku z zapętlającym się ładowaniem faktów. Niestety nie sprawdzałem co się dzieje na fejsbuku

Reply:
Odnosi się do powyższego ifa. Może być tak, że będziemy zmuszeni obsługiwać oznaczanie w doładowanych elementach strony (np. na FB), więc wtedy trzeba będzie sprawdzić, czy działa FB, ale także czy wielokrotne oznaczanie z powrotem się nie wywala. Dopiszę tu komentarz, żeby stestować to względem issue, które rozwiązywałeś.

Reply:
ok